### ISSUE TITLE
Discrepancy in CIRR test set

### ISSUE THREAD

**Issue by hmchuong** [lucas-ventura/CoVR#18]
Hi,

I checked this [#15](https://github.com/lucas-ventura/CoVR/issues/15#issuecomment-2104672587) updated results but cannot reproduce the similar numbers on the CIRR test set although my validation set number is very similar. 

I used this evaluation script and submit the files to the server:
```
python test.py test=cirr model/ckpt=cirr_ft-covr+gt
```

My numbers is

Method|R@1 | R@2 | R@5 | R@10 | R@50 | Recall_subset @ 1 | Recall_subset @ 2 | Recall_subset @ 3
--|-- | -- | -- | -- | -- | -- | -- | --
Zero-shot|27.566 | 38.265 | 52.506 | 63.494 | 85.976 | 71.277 | 86.217 | 94.048
Train|38.241 | 51.205 | 67.518 | 78 | 94.217 | 77.277 | 91.036 | 96.337

My number on the validation set

Method|R@1 | R@5 | R@10 | R@50 | Recall_subset @ 1 | Recall_subset @ 2 | Recall_subset @ 3
--|-- | -- | -- | -- | -- | -- | --
Zero-shot|29.06 | 55.2 | 66.13 | 87.54 | 72.73 | 87.39 | 93.51
Train|41.55 | 70.08 | 79.91 | 94.16 | 78.51 | 91.75 | 95.93

Could you provide your json files for CIRR test?

**Comment by lucas-ventura** [comment]
Hi,

I just checked again and I got the results from [here](https://github.com/lucas-ventura/CoVR/issues/15#issuecomment-2104672587).

To ensure we're on the same page, can you please do ``git pull``?

Then, run the following command to test the CIRR checkpoint:
```bash
python test.py test=cirr model/ckpt=cirr_ft-covr+gt
```
This should produce an R@1 value of 50.41.

You can also try running the test evaluation with the webvid-covr checkpoint:
```bash
python test.py test=cirr model/ckpt=webvid-covr
```
This should result in an R@1 of approximately 39.277.

After running these commands, please let me know the results you get. Additionally, could you verify if the ``blip-embs-larg`` directory contains 2315 blip embeddings?

**Comment by hmchuong** [comment]
My `blip-embs-larg` contains 8082 blip embeddings, that's the problem. I can reproduce the numbers now.
Thanks

**Comment by Jasmine-tjy** [comment]
> My `blip-embs-larg` contains 8082 blip embeddings, that's the problem. I can reproduce the numbers now. Thanks

Dear hmchuongs, I noticed that I have the same issue as you mentioned above, my blip-embs-larg also contains 8082 embeddings. But I am quite confused how to get 2315 blip embeddings for this CIRR test, since the test1 file contains 8082 images and without any subfolders.
Looking forward to your reply :-D
